nautobot_ssot/integrations/servicenow/signals.py

```diff
@@ -0,0 +1,50 @@
+# pylint: disable=duplicate-code
+"""Signal handlers for ServiceNow integration."""
+
+from nautobot.core.signals import nautobot_database_ready
+from nautobot.extras.choices import CustomFieldTypeChoices
+from nautobot.utilities.choices import ColorChoices
+
+
+def register_signals(sender):
```

Comment:
Where is this meant to be called from? I don't see it in the diff.

Reply:
It's in the previous, already merged PR: https://github.com/nautobot/nautobot-plugin-ssot/blob/develop/nautobot_ssot/__init__.py#L55